arviz/plots/energyplot.py

```diff
@@ -5,8 +5,8 @@
 from ..utils import get_stats
 
 
-def energyplot(trace, kind='kde', bfmi=True, figsize=None, legend=True, shade=(1, .75),
-               color_shade=('C0', 'C5'), bw=4.5, skip_first=0, kwargs_shade=None, ax=None,
+def energyplot(trace, kind='kde', bfmi=True, figsize=None, legend=True, fill_alpha=(1, .75),
```

Comment:
Not sure about using alpha, because for some functions alpha is related to credible intervals. Maybe we should give preference to the matplotlib...

Reply:
That's a good point, but maybe giving preference to matplotlib is right here:  in a few of these functions, passing in `alpha` will go to the `**kwargs`, and then get passed to a matplotlib function (in particular, I had renamed this to `alpha` before, and got an error for reusing an argument).